cmd/config.go

```diff
@@ -304,19 +304,17 @@ func addPathsFromDirectory(ctx context.Context, stderr io.Writer, a *appState, d
 	return nil
 }
 
-// ConfigToWrapper converts the Config struct into a ConfigOutputWrapper struct
-func ConfigToWrapper(config *Config) *ConfigOutputWrapper {
-	cfgw := &ConfigOutputWrapper{Global: config.Global, Paths: config.Paths}
-	var providers []*ProviderConfigWrapper
+// Wrapped converts the Config struct into a ConfigOutputWrapper struct
+func (config *Config) Wrapped() *ConfigOutputWrapper {
```

Comment:
I like having this as a method on Config vs. passing the Config struct into a function like we had it before.